### PR TITLE
Fix bug switch

### PR DESCRIPTION
**Pull request by mohammadijavad**

### CODE REVIEW

File: app/[locale]/page.tsx
Comment (amirkh330) [inline app/[locale]/page.tsx:1]: clean this page after testing your component

File: components/ComboBox/ComboBox.constants.ts [hunk 1]
```diff
@@ -1,8 +1,4 @@
-export interface Option {
-  content: string
-  id: string
-}
-
+import { Option } from './ComboBox.types'
 export const options: Option[] = [
```
Comment (amirkh330) [inline components/ComboBox/ComboBox.constants.ts]: don't use constant variables in components, get data outside components as props

File: components/ComboBox/ComboBox.tsx [hunk 2]
```diff
@@ -1,84 +1,54 @@
...
-    }
-
+import useComboBox from './ComboBox.biz';
+import { ComboBoxProp } from './ComboBox.types';
+import { useEffect } from "react";
+const ComboBox: React.FC<ComboBoxProp> = (props) => {
+    const { options, placeHolder, type, id, size,
+        outlineInp, defaultValue, disabled, inputValue,
+        notfound, open, openOnFocus, readonly, sx } = props
+    // hook handel select item and show modal
```
Comment (amirkh330) [inline components/ComboBox/ComboBox.tsx]: remove unnecessary comment

File: components/ComboBox/ComboBox.tsx [hunk 3]
```diff
@@ -1,84 +1,54 @@
...
+    const { inputRef, dropdownRef, openListHandler, isOpen, searchTerm,
+        selectedOption, selectOptionHandler, setSearchTerm,
+        serchOptionsHandler, onblurHandler } = useComboBox()
+    // set data
```
Comment (amirkh330) [inline components/ComboBox/ComboBox.tsx]: remove unnecessary comment

**Review by hesamhabibih — APPROVED** [review]

File: components/ComboBox/ComboBox.types.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  readonly?: boolean
+  open?: boolean
+  openOnFocus?: boolean
+  notfound?: string
+  inputValue?: string
+  id: string
+  sx?: Properties<string | number, string & {}>
+  outlineInp?: 'indigo' | 'red' | 'blue' | 'green' | 'Pink'
+  placeHolder: string
+  type: 'text' | 'number' | 'email'
```
Comment (amirkh330) [inline components/ComboBox/ComboBox.types.ts]: add customCss

**Pull request closed without merging**